bin/hdfcoinc/pycbc_multiifo_add_statmap

```diff
@@ -113,11 +113,12 @@ logging.info('Copying foreground datasets')
 for k in files[0]['foreground']:
     if not k.startswith('fap') and k not in all_ifos:
         pycbc.io.combine_and_copy(f, files, 'foreground/' + k)
-
-logging.info('Copying background datasets')
-for k in files[0]['background']:
-    if k not in all_ifos:
-        pycbc.io.combine_and_copy(f, files, 'background/' + k)
+injection_style = args.background_files != None
```

Comment:
it might be preferable to put this right after argument parsing (rather than in the middle of two parallel steps) since it's a flag with global effect. otherwise looks fine